raster/src/main/scala/geotrellis/raster/io/geotiff/reader/TiffTagsReader.scala

```diff
@@ -14,16 +14,13 @@ import java.nio.{ ByteBuffer, ByteOrder }
 
 
 object TiffTagsReader {
-  def read(path: String): TiffTags = read(Filesystem.slurp(path))
 
-  def read(bytes: Array[Byte]): TiffTags = {
-    val byteBuffer = ByteBuffer.wrap(bytes, 0, bytes.size)
-
-    // Set byteBuffer position
-    byteBuffer.position(0)
+  def read(path: String): TiffTags = read(Filesystem.streamByteBuffer(path))
+  
+  def read(byteBuffer: ByteBuffer): TiffTags = {
 
     // set byte ordering
-    (byteBuffer.get.toChar, byteBuffer.get.toChar) match {
+    (byteBuffer.reader.toChar, byteBuffer.reader.toChar) match {
```

Comment:
That was code that was left over from my original implementation. I though I had removed it. Sorry about that.